openpype/hosts/blender/blender_addon/addons/update_paths_to_last_version.py

```diff
@@ -0,0 +1,213 @@
+import os
+import logging
+import re
+
+from openpype.pipeline.anatomy import Anatomy
+from openpype.lib import StringTemplate
+from openpype.pipeline.context_tools import get_template_data_from_session
+
+import bpy
+
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+bl_info = {
+    "name": "Update paths to last version",
+    "description": "Update all concerned paths in scene to automatically target the last version existing",
+    "author": "Quad",
+    "version": (1, 0),
+    "blender": (2, 80, 0),
+    "category": "Render",
+    "location": "View 3D > UI",
+}
+
+
+class VIEW3D_PT_UPDATE_PATHS(bpy.types.Panel):
+    bl_label = "Select object types to update"
+    bl_space_type = "VIEW_3D"
+    bl_region_type = "UI"
+    bl_category = "Quad"
+
+    update_alembics = bpy.props.BoolProperty(default=True)
+    update_shader_files = bpy.props.BoolProperty(default=True)
+
+    def draw(self, context):
+        layout = self.layout
+        col = layout.column(align=True)
+        col.operator("paths.update_animation_paths", text="Update alembics to animation", icon="MESH_CUBE").mode = "animation"
+        col.operator("paths.update_animation_paths", text="Update alembics versions", icon="MESH_CUBE").mode = "update_cache_versions"
+
+
+class OBJECT_OT_UPDATE_PATHS(bpy.types.Operator):
+    bl_idname = "paths.update_animation_paths"
+    bl_label = "Update tasks in scene objects paths"
```

Comment:
Not sure we are updating tasks in path in this class

Reply:
Fixed in the next commit